grails-app/views/inventoryItem/_showStockHistory.gsp

```diff
@@ -298,12 +298,7 @@
                                         <img src="${resource(dir: 'images/icons/silk', file: 'arrow_refresh_small_blue.png' )}" title="${format.metadata(obj:TransactionScope.INTERNAL)}"/>
                                     </g:if>
                                     <g:elseif test="${stockHistoryEntry?.transaction?.transactionType?.isAdjustment()}">
-                                        <g:if test="${stockHistoryEntry?.isCredit}">
-                                            <img src="${resource(dir: 'images/icons/silk', file: 'arrow_up.png')}" title="${format.metadata(obj:stockHistoryEntry?.transaction?.transactionType)}"/>
-                                        </g:if>
-                                        <g:else>
-                                            <img src="${resource(dir: 'images/icons/silk', file: 'arrow_down_red.png')}" title="${format.metadata(obj:stockHistoryEntry?.transaction?.transactionType)}"/>
-                                        </g:else>
+                                        <img src="${resource(dir: 'images/icons/silk', file: 'arrow_up_green_down_red.png')}" title="${format.metadata(obj:stockHistoryEntry?.transaction?.transactionType)}"/>
```

Comment:
There are two ways to do this:
1) Always show the up/down arrow for all adjustments, even if there are only credits or only debits, and only show it for the first entry of the adjustment transaction. This is how it's currently implemented.
![Screenshot from 2025-05-21 13-34-50](https://github.com/user-attachments/assets/5e0928e5-d59b-4a89-b767-6f1d7edd3469)

2) Have adjustments show an icon for all entries of the transaction and have that icon either be an up or down arrow depending on if it's a positive/negative adjustment
![image](https://github.com/user-attachments/assets/ea51ca84-cff8-4f85-b580-020d50bb7d9e)

Both are easy to do, so we should decide which we prefer

Reply:
Awesome. I like the first one better.

Reply:
I like the first one too. Is it just me, or the red arrow is a bit higher than the green one? :thinking:

Reply:
I think they're alligned :smile: 

![image](https://github.com/user-attachments/assets/26a31a82-bb6b-433e-87ed-a8782bd32563)